code/datums/outfits/ert/fsf_ert.dm

```diff
@@ -0,0 +1,124 @@
+/datum/outfit/admin/ert/fsf_ert
+    name = "Free Solarian Fleets Marine"
+
+    uniform = /obj/item/clothing/under/rank/fatigues/marine
+    shoes = /obj/item/clothing/shoes/swat/ert
+    gloves = /obj/item/clothing/gloves/swat/ert
+    belt = /obj/item/storage/belt/military
+    back = /obj/item/storage/backpack/satchel_norm
+    head = /obj/item/clothing/head/helmet/space/void/sol
+    suit = /obj/item/clothing/suit/space/void/sol
+    suit_store = /obj/item/gun/projectile/automatic/rifle/sol
+    id = /obj/item/card/id/syndicate
```

Comment:
The syndic card should only be used by "proper" antags as it allows self-modification.
This is not needed/wanted for a ert team.